### PR TITLE
Update wpmjson schema

### PR DESCRIPTION
**Pull request by thelovekesh**
- Remove team and introduce author
- Update validator for strict validation
- Add test cases to test edge cases
- Add json schema for wpm.json config file

### CODE REVIEW

File: pkg/pm/wpmjson/wpmjson.go [hunk 1]
```diff
@@ -89,26 +89,31 @@ func (c *Config) Validate() error {
 	if c.License != "" {
 		errs.Add("license", validator.IsValidLicense(c.License))
 	}
+	if c.Author != "" {
+		errs.Add("author", validator.IsValidAuthor(c.Author))
+	}
```
[the remark this answers is not in the document]
Comment (thelovekesh) [reply pkg/pm/wpmjson/wpmjson.go:94]: it will only reach this point if private is not set of false
